lib/cpp/src/thrift/server/TNonblockingServer.cpp

```diff
@@ -1524,7 +1532,7 @@ void TNonblockingIOThread::breakLoop(bool error) {
   }
 
   // sets a flag so that the loop exits on the next event
```

Comment:
I guess the comment is stale now. The loop now exits once the current iteration is complete.